server/routes/dexcom.js

```diff
@@ -0,0 +1,14 @@
+const fetch = require('node-fetch');
+
+function dexReq(req) {
+  fetch(`https://sandbox-api.dexcom.com/v2/users/self/egvs?startDate=${req.query.start}&endDate=${req.query.end}`, {
+          method: 'get',
+          headers: {
+            'Authorization': 'Bearer ' + req.user.dexcomAccessToken
+          },
+      })
+      .then(res => res.json())
```

Comment:
[Dexcom's docs](https://developer.dexcom.com/authentication) mention that we should handle a 401 response code, which means that the access token expired and we need to use the user's refresh token to obtain a new one